lib/dk_buttons_and_leds/dk_buttons_and_leds.c

```diff
@@ -101,8 +106,19 @@ static void buttons_scan_fn(struct k_work *work)
 {
 	static uint32_t last_button_scan;
 	static bool initial_run = true;
+	static bool irq_enabled;
 	uint32_t button_scan;
 
+	if (irq_enabled) {
+		/* Disable GPIO interrupt */
+		int err = callback_ctrl(false);
+
```

Comment:
Nit: let's keep the pattern of 'err' and 'if (err)' with no empty line in between